src/Main.js

```diff
@@ -0,0 +1,16 @@
+import { Component } from 'react';
+import HornedBeasts from './HornedBeasts.js';
+
+class Main extends Component {
+    render() {
+        return (
+        <div>
+        <HornedBeasts title="rhino" imageUrl="https://i.imgur.com/zn7AI5W.jpg" description="big chonker" />
+        <HornedBeasts title="goat" imageUrl="https://i.imgur.com/aIf3Sez.jpg" description="goat is smug note: horns hard to see" />
```

Comment:
Same here, we can indent these 2 `<HornedBeasts />` components 2 spaces 
Fun tip, once we get a lot of props, it helps to indent those under the component for easier readability 👍